src/executePrettier/buildPrettierOptions.test.js

```diff
@@ -109,40 +107,46 @@ it('uses vue as the parser if current scope is listed as a Vue SFC scope in sett
   expect(actual).toEqual({ parser: 'vue' });
 });
 
-it('does not use editorconfig options if that setting is not enabled', () => {
+it('passes editorconfig option to prettier', () => {
+  let editorconfig = false;
+
   const editor = buildMockEditor();
+  const mockPrettierInstance = { resolveConfig: { sync: jest.fn() } };
+  getPrettierInstance.mockImplementation(() => mockPrettierInstance);
+  shouldUseEditorConfig.mockImplementation(() => editorconfig);
+  getCurrentFilePath.mockImplementation(() => 'parent-dir/foo.js');
   getPrettierOptions.mockImplementation(() => ({}));
-  shouldUseEditorConfig.mockImplementation(() => false);
-
   buildPrettierOptions(editor);
+  expect(mockPrettierInstance.resolveConfig.sync.mock.calls[0][1]).toEqual({ editorconfig });
 
-  expect(buildEditorConfigOptions).not.toHaveBeenCalled();
+  editorconfig = true;
+  buildPrettierOptions(editor);
+  expect(mockPrettierInstance.resolveConfig.sync.mock.calls[1][1]).toEqual({ editorconfig });
 });
```

Comment:
Can we either:

1. split this into two tests (one for `true` and one for `false`)
2. just test one of the two cases since it shouldn't make a difference whether it's `true` or `false`

(I'd prefer the latter). This way we aren't mutating a variable and doing multiple expectations in a single test.

Reply:
Makes sense.  I appreciate you explaining "why" - I honestly am not a heavy tester so I don't have much knowledge of conventions or best practices.